xla/translate/mhlo_to_hlo/tests/export.mlir

```diff
@@ -585,6 +585,20 @@ func.func @callee(%arg0: tensor<4xi32>, %arg1: tensor<4xi32>) -> (tensor<4xi32>,
 
 // -----
 
+// CHECK:  HloModule
+func.func @main(%arg0: tensor<128x32xf32>) -> tensor<128x32xf32> {
+  %0 = "mhlo.collective_broadcast"(%arg0) {
+    replica_groups = dense<[[0, 1], [2, 3]]> : tensor<2x2xi64>,
+    channel_handle = #mhlo.channel_handle<handle = 1, type = 0>
+  } : (tensor<128x32xf32>) -> tensor<128x32xf32>
+  func.return %0 : tensor<128x32xf32>
+}
+// CHECK:  ENTRY
+// CHECK:  [[ARG:%.*]] = f32[128,32] parameter(0)
+// CHECK:  ROOT [[RESULT:%.*]] = f32[128,32] collective-broadcast(f32[128,32] [[ARG]]), channel_id=1, replica_groups={{\{\{}}0,1},{2,3}}
```

Comment:
Something seems off about this nesting.. `{{\{\{}}0,1},{2,3}}`


Can we instead use `// CHECK-SAME{LITERAL}:  replica_groups={{0,1},{2,3}}` similar to the TP 3 below this one.

Reply:
Yeah, that was ported from the collective-permute test just below where it also has strange formatting for `source-target-pairs`. I'll change it.

Reply:
Done